tests/e2e/project/patches.spec.ts

```diff
@@ -37,11 +37,35 @@ test("navigate patch details", async ({ page }) => {
 test("use revision selector", async ({ page }) => {
   await page.goto(`${cobUrl}/patches/0f3697fed2743549e3bf531e9fa81284a6de1466`);
   await page.locator("role=link[name='Files']").click();
-  await page.locator("text='Revision febcbbd'").click();
-  await page.locator("role=link[name='Revision febcbbd']").click();
+
+  // Validating the latest revision state
+  await expect(
+    page.locator("role=cell[name='Had to push a new revision']"),
```

Comment:
You can also write this as follows:
```suggestion
    page.getByRole("cell", { name: "Had to push a new revision" }),
```

Reply:
Thanks @geigerzaehler for the feedback!
I followed the style of the codebase and kept using the `locator("role=...)`

Do you know if there is any benefit of using one over the other?
I also like `getByRole` better since that is what the `Pick locator` in debug mode outputs.

Reply:
Ok nvm, from what I found in the playwright library it's a wrapper over the bare `locator` function

Reply:
> Do you know if there is any benefit of using one over the other?

For me there are two main benefits: It’s easier to read since the information is not encoded with a custom syntax in a string but just regular JavaScript. And then it is also easier to write correctly because I get completion support and the code is type checked.

On top of that the `getByXXX()` functions also offer more [advanced features](https://playwright.dev/docs/api/class-framelocator#frame-locator-get-by-role-option-checked) for more expressive locators.

Reply:
✅